Translate docs to english (#40)
digipost/signering-docs: source/adressering-av-undertegner.rst

.. _addressing-of-signer:

Addressing of signer
***************************
Signers that are to sign in :ref:`portal flow <signing-in-portal-flow>` may be addressed in two different ways:

1. By email and/or SMS (only available for private organizations)
2. By national identity number

..  WARNING::
    How you address a signer affects the :ref:`content and appearance of the signer's signature <identify-signers>`.

> 1. Address using e-mail / SMS

Reviewer: ```suggestion
1. Addressing signers by e-mail / SMS
```

===============================

.. NOTE::
   This option is only available for private organizations

> On addressing by email/SMS, the signature request is not linked to a national identity number. We therefore do not have any way of verifying that the correct person is reading and signing. It is therefore not necessary to log in so as to read and sign. The recipient receives an email and/or SMS with a unique code and link to access the request. View :ref:`notifications <notifications-without-national-identity>` to see what the notification looks like.

Reviewer: ```suggestion
When addressing signers by email/SMS, we will not be able to link signers to their national identity number. Therefore, we do not have any way to verify that the correct person is reading and signing. The recipients receive an email and/or SMS with a unique code and link to access the document(s) to sign, instead of being required to log in. See :ref:`notifications <notifications-without-national-identity>` to review what the notifications look like.
```

1. Addressing signers by e-mail / SMS
===============================

.. NOTE::
   This option is only available for private organizations

On addressing by email/SMS, the signature request is not linked to a national identity number. We therefore do not have any way of verifying that the correct person is reading and signing. It is therefore not necessary to log in so as to read and sign. The recipient receives an email and/or SMS with a unique code and link to access the request. View :ref:`notifications <notifications-without-national-identity>` to see what the notification looks like.

..  IMPORTANT::
    The signature itself is just as secure and valid as when you address using national identity number, but you as the sender are responsible for ensuring that the correct person reads and signs.

In the case of e-mail/SMS addressing, the national identity number will not be included in the signed document for privacy reasons. Name and any date of birth will continue to be included in the signed document.

See :ref:`Signing in portal flow with addressing by e-mail / SMS <signing-in-portal-flow-without-national-identity>` for more information and visual examples.

> 2. Address using national identity number

Reviewer: ```suggestion
2. Addressing signers by their national identity number
```

================================

> To address using a national identity number, you must know the signer's national identity number. Addressing using a national identity number is the safest way to reach the recipient, since it requires logging in with an electronic ID in order to read and sign.

Reviewer: Snur om litt fordi setningen virket litt sånn åpenbar slik den var skrevet fra før (for å bruke noe informasjon må du vite informasjonen liksom.)

```suggestion
If you know the national identity numbers of the signers, you can address them using their identity numbers. This is the most secure way to reach the expected recipients, since it requires them to log in with an electronic ID in order to read and sign.
```

.. _addressing-of-signer:

> Addressing of signer

Reviewer: Gjøre om dette til flertall? 

```suggestion
Addressing signers
```

(Sikkert da greit å samtidig gjøre om referansen til `_addressing-signers`)

***************************

> Signers that are to sign in :ref:`portal flow <signing-in-portal-flow>` may be addressed in two different ways:

Reviewer: Lurte på å kanskje endre dette til mer aktiv form, sånn?

```suggestion
You may address signers for :ref:`portal flow <signing-in-portal-flow>` in two different ways:
```

Synes liksom "Signers that are to sign..." hørtes litt rart ut, men mulig det er helt innafor altså.

1. By email and/or SMS (only available for private organizations)
2. By national identity number

> ..  WARNING::

Reviewer: Er ikke dette noe sånn semantikk-markup? Blir ikke dette til noe sånn boks med uttropstegn eller et eller annet sånt? (Er det intentional å endre fra `CAUTION::` til `WARNING::`?)

Author: Det blir til en boks ja, så da har det sikkert noe å si hva det står der ja

>     How you address a signer affects the :ref:`content and appearance of the signer's signature <identify-signers>`.

Reviewer: ```suggestion
    How you choose to address the signer(s) affects the :ref:`content and appearance of the signature(s) in the resulting signed document <identify-signers>`.
```

2. Address using national identity number
================================
To address using a national identity number, you must know the signer's national identity number. Addressing using a national identity number is the safest way to reach the recipient, since it requires logging in with an electronic ID in order to read and sign.

Even if the signer is addressed using a national identity number, the signer will be notified and will receive a link to log in by email or SMS – see :ref:`notification and contact details <notifications>` for more information..

..  TIP::
    See :ref:`signing in portal flow with national identity number <signing-in-portal-flow-with-national-identity>` for more information and visual examples.

.. _notifications:

Notifications and contact details
---------------------------------

When you select signing in portal flow, emails and/or SMS notifications are sent to the signers.

 - All signers must have at least one email address and mobile number.
 - Sending an SMS is voluntary and can be ordered by the service owner, at a cost of NOK 40 cents per SMS.
 - If a signer has a mobile number and not an email address, an SMS will always be sent.

>  - The service only supports Norwegian mobile numbers.

Author: Støtter vi utenlandske numre nå? eller er det bare i portal?

Reviewer: Mener vi skal støtte det i både API og portal.

Using the Contact and Reservation Register
============================================

Further details concerning the use of the Contact and Reservation Register

On sending out subsequent notifications (either deferred activation due to chained signature, or reminders) a new lookup is made in the register to retrieve the latest updated contact details.

If the Lookup Service for the Contact and Reservation Register is unavailable when reminders are sent, the result of the lookup on creating the request will be used.

Opt-out concerning deferred initial notifications: In the scenario where the service owner has set a chained sequence for the signers, and the initial notification is to be sent to a signer who, in the period between creation of the request and sending the initial notification, has opted out of electronic communication, the entire request will fail.

Opt-out concerning reminders: If the end user has opted out after the request was created, but the request has already been activated, no reminders (email/text message) will be sent, but the request will not fail either until any expiry of the signing deadline.

> Requests with overridden contact details with foreign mobile numbers will be rejected, while foreign mobile numbers from the Contact and Reservation Register will be ignored.

Author: Se kommentar over 👆

Reviewer: Samme her. Dette er nok ikke tilfelle lenger.
